swx.d/10_swx_secrets

```diff
@@ -15,6 +15,59 @@ swx_secrets_encrypt ()
   trousseau set "file:$1" "$(openssl enc -base64 -A -in ${devops}/$1)"
 }
 
+swx_secrets_install ()
+{
+    # check if trousseau already installed
+    trousseau --version > /dev/null 2>&1
```

Comment:
Not sure if there's a cleaner way to check if trousseau's already installed.

Reply:
Yeah, I would probably use `which trousseau > /dev/null 2>&1` which returns a 0 if there is no trousseau command in the current `PATH`.